docs/index.md

```diff
@@ -0,0 +1,174 @@
+# Getting Started:
+
+```
+$ poetry install
+$ poetry shell
+$ python -m bucket.example
+```
+
+This page describes how to use Bucket to create coverpoints, sample data and merge results. Examples of coverpoints in use can be seen in `example.py`.
+
+## Coverpoints
+
+A coverpoint consists of one or more axes, which are then crossed. Each possible combination of the axes values is called a bucket. Each bucket has a default goal of 10 hits, which can be modified as required, or can be made illegal or ignored. 
```

Comment:
`Each bucket has a default goal of 10 hits` ->  `Each bucket has a default goal with a target of 10 hits`